static/src/javascripts-legacy/projects/common/modules/experiments/tests/acquisitions-love-boat.js

```diff
@@ -0,0 +1,143 @@
+define([
+    'bean',
+    'common/modules/commercial/contributions-utilities',
+    'common/utils/template',
+    'common/utils/config',
+    'common/utils/$',
+    'common/views/svgs',
```

Comment:
in that case, I think https://github.com/guardian/frontend/blob/master/docs/03-dev-howtos/04-inline-svgs.md needs an update :)